express-hello-world/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const app = express();
+app.use(function (req, res, next) {
+  // console.log('Time:', Date.now());
+  next();
+  res.send('Hello');
```

Comment:
We don't want to combine `res.send` and `next` should always be one or the other. In this case we just want `res.send` here without the `next` call. We will discuss in more detail in the Q and A.